datafusion/physical-expr-common/src/sort_expr.rs

```diff
@@ -117,6 +117,16 @@ impl PhysicalSortExpr {
         self.options.nulls_first = false;
         self
     }
+
+    /// Like `fmt_sql` in `PhysicalExpr`, prints a [`PhysicalSortExpr`] in a SQL-like format.
```

Comment:
If you put the whole thing in `[]` then rustdoc will make an automatic link 

```suggestion
    /// Like [`PhysicalExpr::fmt_sql`] prints a [`PhysicalSortExpr`] in a SQL-like format.
```